templates/base.html

```diff
@@ -17,7 +17,7 @@
         {% else %}
         <meta name="author" content="{{ AUTHOR }}" />
         {% endif %}
-
+        {% from '_includes/_defaults.html' import READING_TIME_LOWER_LIMIT with context %}
```

Comment:
Why import variable from default here? It should be imported in the post_stats file; right above where it is used.

Reply:
Changed in next push, can you recheck?